packages/@lwc/engine/dom/src/index.ts

```diff
@@ -5,6 +5,32 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-export * from './framework/main';
+// Polyfills ---------------------------------------------------------------------------------------
+import './polyfills/proxy-concat/main';
+import './polyfills/aria-properties/main';
 
-export * from '../../src';
+// TODO [#0]: We need to address the problem where engine-core needs the above polyfills.
```

Comment:
This would make sense to remove the TODO from here and move it to the book of work document.